### DISCUSSION TITLE
Práctica

### DISCUSSION THREAD

**Discussion by JeanCCS1** [Launch-X-Latam/MisionFrontEnd#104] [Q&A]
Que práctica debemos de realizar esta semana que me perdí en eso

**Comment by JulioRamnses** [comment]
Tendremos que hacer la practica de Abogadobot que se encuentra en la introducción de frontend

**Reply by JosafatJimenezB** [reply]
asi es

**Reply by FaraN1ght** [reply]
Cómo están chicos, acabo de entrar al proceso y estoy un poco perdido... se debe armar algún entregable de la introducción, en dónde y hasta cuándo?

**Reply by JosafatJimenezB** [reply]
si, debes ir realizando las practicas poco a poco, ahorita estamos con lo de abogabot el cual debes subirlo a un repo para que al final de la misión lo entregues y pases a la de backend
